Remove -march= such that consumers of argon2 can add a more specialized `-march=...` to their profile. 

--- Makefile
+++ Makefile
@@ -49,7 +49,7 @@
 				-Wextra -Wno-type-limits -Werror -coverage -DTEST_LARGE_RAM
 
 OPTTARGET ?= native
-OPTTEST := $(shell $(CC) -Iinclude -Isrc -march=$(OPTTARGET) src/opt.c -c \
+OPTTEST := $(shell $(CC) -Iinclude -Isrc src/opt.c -c \
 			-o /dev/null 2>/dev/null; echo $$?)
 # Detect compatible platform
 ifneq ($(OPTTEST), 0)
@@ -57,7 +57,7 @@
 	SRC += src/ref.c
 else
 $(info Building with optimizations for $(OPTTARGET))
-	CFLAGS += -march=$(OPTTARGET)
+	#CFLAGS += -march=$(OPTTARGET)
 	SRC += src/opt.c
 endif
 
